### DISCUSSION TITLE
If Http Add On is installed but no one is using it, should it impact anything?

### DISCUSSION THREAD

**Discussion by Leonardo-Ferreira** [kedacore/http-add-on#850] [Q&A]
Pretty simple: night of day zero, we installed the HTTP Add On, the following day we had a mysterious slowness problem and had to scale the cluster about 40%... at the time we didn't think the HTTP add-on could be causing the issue...

So my question is: if the add-on is installed but no app is using it, should it cause any impact?!?

**Comment by linkvt** [comment]
The Add-on should definitely not cause anything like this, as it only does something when a) HTTPScaledObjects are changed or b) requests pass through the interceptor.

Closing this discussion as I'm doing a cleanup right now, if this is a bug that still exists please create an issue, otherwise just let us know.
